CONTRIBUTING.md

```diff
@@ -49,3 +49,12 @@ branches.
 
 Please see our [git style guide](https://github.com/alphagov/styleguides/blob/master/git.md)
 which describes how we prefer git history and commit messages to read.
+
+### Review apps
+
+When a pull request is opened Heroku may create a [review app](https://devcenter.heroku.com/articles/github-integration-review-apps#viewing-review-apps)
```

Comment:
```suggestion
When a pull request is opened, Heroku may create a [review app](https://devcenter.heroku.com/articles/github-integration-review-apps#viewing-review-apps)
```

Reply:
I wonder should we say "should" instead of "may," if Heroku not-creating-the-app is unexpected behaviour.

Reply:
The behaviour depends on who is creating the pull request; if you are not part of the alphagov org then Heroku not creating the app is expected behaviour.

I wasn't sure if it was worth making that clear in this document though, what do you think?

Reply:
Ah I see - thanks. "May" sounds fine then! You're right, they probably don't need to know that.